프로그래머스/1/12982. 예산/예산.js

```diff
@@ -0,0 +1,13 @@
+function solution(d, budget) {
+    const amountArr = d.slice().sort((a, b) => a - b);
+    let balance = budget;
+    let count = 0;
+    
+    for(let i = 0; i < amountArr.length; i++) {
+        if(balance - amountArr[i] >= 0) {
+            balance = balance - amountArr[i];
```

Comment:
```suggestion
            balance -= sortedRequests[i];
```
이런식으로 간결하게 계산해도 좋을 것 같네용 !!